.github/workflows/gradle.yml

```diff
@@ -0,0 +1,74 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will build a Java project with Gradle and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-gradle
+
+name: Java CI with Gradle
```

Comment:
You'll want a deploy run on a tagged release to upload the distribution to the release artifacts.

Reply:
Let's defer this until we have a version that is release-ready.